lib/avromatic/model/types/record_type.rb

```diff
@@ -36,10 +36,6 @@ def coercible?(input)
           false
         end
 
-        def coerced?(value)
-          value.nil? || value.is_a?(record_class)
-        end
-
```

Comment:
Are you sure about that? This one is different from the `AbstractType#coerced?`

Reply:
yes
```
        def coerced?(value)
          value.nil? || value_classes.any? { |value_class| value.is_a?(value_class) }
        end
```
`value_classes = [record_class].freeze`

Reply:
I can restore it back if you think it should be out of this PR